src/AgentHeader.php

```diff
@@ -64,6 +64,7 @@ public static function buildAgentHeader($headerInfo)
         //      - gapicVersion (gapic/)
         //      - apiCoreVersion (gax/)
         //      - grpcVersion (grpc/)
+        //      - restVersion (rest/)
```

Comment:
Don't forget to add this to the `$headerInfo` phpdoc for the method!

Reply:
Done.